flask/app/routes.py

```diff
@@ -162,3 +163,42 @@ def change_password():
     except:
         db.session.rollback()
         return 'Server error', 500
+
+
+@app.route('/api/participants/<participant_id>', methods = ['PATCH'])
+# @login_required # check document for how to test endpoints with login signing? 
+def update_participants(participant_id):
+    patch_json = request.get_json()
+
+    # convert to dictionary
+    patch_d = json.loads(patch_json)
+
+    # get the PK
+    id = patch_d['participant_id']
+
+    # check if PK exists, is this necessary ??
+    q = db.session.query(models.Participant.participant_id)\
+        .filter_by(participant_id = id)
```

Comment:
Yes, you should check if the primary key exists and respond with a 404 Not Found appropriately.